dali/base/dadfs.hpp

```diff
@@ -494,6 +497,10 @@ class DistributedFilePropertyLock {
     {
         return file->queryAttributes();
     }
+    IPropertyTree &queryHistory()
+    {
+        return file->queryHistory();
+    }
```

Comment:
I use this function to update target file History records (Copy them from the source and and for a new about the current operation. So, can I do same if I use lock.queryFile().queryHistory()?